BUG: StatusResponse

## Descriere
În varianta sandbox a API-ului Netopia, răspunsul de la endpoint-ul `/operation/status` conține un câmp suplimentar `card`, care nu este definit în clasa `StatusResponse` din SDK-ul Python.

Acest lucru duce la următoarea eroare la parsarea răspunsului JSON:

[ERROR] Failed to parse response into StatusResponse:
StatusResponse.init() got an unexpected keyword argument 'card'

### Soluție propusă

Se poate adăuga următorul câmp în clasa `StatusResponse`:

```python
card: Optional[Dict] = None

--- a/netopia_sdk/responses/models.py
+++ b/netopia_sdk/responses/models.py
@@ -55,6 +55,7 @@ class StatusResponse(ApiResponse):
     order: Optional[Dict] = None
     payment: Optional[PaymentResponseData] = None
     customerAction: Optional[Dict] = None
+    card: Optional[Dict] = None
 
 
 @dataclass

